coilsnake/assets/modulelist.txt

```diff
@@ -22,5 +22,6 @@ eb.BattleBgModule
 eb.CompressedGraphicsModule
 eb.EnemyModule
 eb.TilesetModule
+eb.TitleScreenModule
```

Comment:
This should be before TilesetModule. TilesetModule takes the longest to compile so all other modules should go before it so they can fail faster.